src/app/aliases/aliases.component.ts

```diff
@@ -17,13 +17,14 @@ export class AliasesComponent extends Base implements OnInit {
   organization$: Observable<Organization | null>;
   collection$: Observable<Collection | null>;
   workflow$: Observable<Workflow | null>;
+  workflowVersionPathInfo$: Observable<WorkflowVersionPathInfo | null>;
   tool$: Observable<DockstoreTool | null>;
 
   public type: string | null;
   public alias: string | null;
   public validType: boolean;
   // Types contains resource types that support aliases
-  public types = ['organizations', 'collections', 'workflows', 'tools', 'containers'];
+  public types = ['organizations', 'collections', 'workflows', 'tools', 'containers', 'workflow-versions'];
```

Comment:
We could use consts for the array values; then the if statements starting at line 42 could at least compare against the consts, and avoid the possibility of a typo. Or there might even be a more elegant solution with enums or something like [this](https://stackoverflow.com/questions/44497388/typescript-array-to-string-literal-type)